extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/thing/thing-types.xml

```diff
@@ -26,5 +27,12 @@
         <description>Selects the color of the light</description>
         <category>ColorLight</category>
     </channel-type>
+    
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
```

Comment:
It nicely gives the Kelvin value here, but in other cases we actually map the color temperature to a Dimmer item, with 0%=cold and 100%=warm. The UI can then directly render nice sliders or buttons for it. This also means that this channel should then be able to handle INCREASE/DECREASE commands. Would you be fine to change this?